### ISSUE TITLE
Windows: _WIN32_WINNT is redefined is a user sets it

### ISSUE THREAD

**Issue by vtorri** [ultravideo/kvazaar#404]
if a user wants to set _WIN32_WINNT when compiling, then this warning appear

```
C:/Documents/msys2/home/vincent.torri/ewpi_64/share/ewpi/packages/libkvazaar/kvazaar-2.3.1/src/encmain.c:41: warning: "_WIN32_WINNT" redefined
   41 | #define _WIN32_WINNT 0x0500
      | 
```

maybe it should be guarded with a #ifndef.

**Comment by Jovasa** [comment]
Yes, I see no reason why it shouldn't be. Considering it has been there for 10 years, nobody has before wanted to define it themselves.

Just wondering how many other defines there are that should have a guard.

**Comment by vtorri** [comment]
fixed in #405
